testnet/stacks-node/src/neon_node.rs

```diff
@@ -96,13 +92,14 @@ enum RelayerDirective {
     ProcessTenure(ConsensusHash, BurnchainHeaderHash, BlockHeaderHash),
     RunTenure(RegisteredKey, BlockSnapshot, u128), // (vrf key, chain tip, time of issuance in ms)
     RegisterKey(BlockSnapshot),
-    RunMicroblockTenure(u128), // time of issuance in ms
+    RunMicroblockTenure(BlockSnapshot, u128), // time of issuance in ms
     Exit,
 }
 
-pub struct InitializedNeonNode {
+pub struct StacksNode {
     config: Config,
     relay_channel: SyncSender<RelayerDirective>,
+    last_sortition_mutex: Arc<Mutex<Option<BlockSnapshot>>>,
     burnchain_signer: BurnchainSigner,
     last_burn_block: Option<BlockSnapshot>,
```

Comment:
What's the difference between `last_burn_block` and `last_sortition_mutex`?

Reply:
(Also, naming peeve: `last_sortition_mutex` doesn't need to tell us it's a mutex-- it's typed as one)

Reply:
`last_sortition_mutex` can be accessed by the relayer thread, whereas `last_burn_block` cannot (it's only available to the main thread).

I think I can just drop `last_burn_block` altogether, since it gets set immediately before `last_sortition_mutex` in practice (the former is done by `.process_burnchain_state()`, whereas the latter is done by `.relayer_sortition_notify()`; both are called one right after the other in the runloop thread).

Reply:
Yep, dropping `last_burn_block` makes sense